Content.Shared/Movement/Systems/SharedJetpackSystem.cs

```diff
@@ -173,11 +181,14 @@ public void SetEnabled(EntityUid uid, JetpackComponent component, bool enabled,
 
         if (enabled)
         {
+            RemComp<WaitingJetpackUserComponent>(user.Value); // DeltaV - Jetpacks automatically turn on when toggled.
             SetupUser(user.Value, uid, component);
             EnsureComp<ActiveJetpackComponent>(uid);
         }
         else
         {
+            if (component.AutomaticMode) // DeltaV - Jetpacks automatically turn on when toggled.
+                EnsureComp<WaitingJetpackUserComponent>(user.Value).Jetpack = uid;
```

Comment:
nit: Consider adding some basic methods to `Content.Shared/_DV/Movement/Systems/SharedJetpackSystem.cs` that encapsulates the `WaitingJetpackUserComponent` logic almost entirely.